assets/src/blocks/Columns/ColumnsEditor.js

```diff
@@ -108,76 +109,78 @@ export const ColumnsEditor = ({ isSelected, attributes, setAttributes }) => {
     const styleClass = getStyleFromClassName(className);
     if (styleClass) {
       setAttributes({
-        columns_block_style: styleClass
+        columns_block_style: styleClass,
       });
     }
   }, [className]);
 
-  const { postType } = useSelect(select => ({
-    postType: select('core/editor').getCurrentPostType()
+  const {postType} = useSelect((select) => ({
+    postType: select('core/editor').getCurrentPostType(),
   }), []);
 
-  const { columnImages } = useSelect(select => {
-    let columnImages = [];
-    columns.forEach(column => {
+  const {columnImages} = useSelect((select) => {
+    // eslint-disable-next-line no-shadow
+    const columnImages = [];
+    columns.forEach((column) => {
       if (column.attachment && column.attachment > 0) {
         const media_details = select('core').getMedia(column.attachment);
         if (media_details) {
-          columnImages[column.attachment] = select('core').getMedia(column.attachment).source_url;
+          columnImages[ column.attachment ] = select('core').getMedia(column.attachment).source_url;
         }
       }
     });
-    return { columnImages };
+    return {columnImages};
   }, [columns]);
 
-  const toAttribute = (attributeName, index) => value => {
+  const toAttribute = (attributeName, index) => (value) => {
     if (['columns_title', 'columns_description'].includes(attributeName)) {
       setAttributes({
-        [attributeName]: value
+        [ attributeName ]: value,
       });
     } else {
-      let columns = JSON.parse(JSON.stringify(attributes.columns));
-      columns[index][attributeName] = value;
-      setAttributes({ columns });
+      // eslint-disable-next-line no-shadow
+      const columns = JSON.parse(JSON.stringify(attributes.columns));
+      columns[ index ][ attributeName ] = value;
+      setAttributes({columns});
     }
-  }
+  };
 
   return (
     <section className={`block columns-block block-style-${columns_block_style} ${className ?? ''}`}>
       {renderEdit(attributes, toAttribute, setAttributes, isSelected)}
       {!isExample &&
-        <header className='articles-title-container'>
+        <header className="articles-title-container">
           <RichText
-            tagName='h2'
-            className='page-section-header'
+            tagName="h2"
+            className="page-section-header"
             placeholder={__('Enter title', 'planet4-blocks-backend')}
             value={columns_title}
             onChange={toAttribute('columns_title')}
             withoutInteractiveFormatting
-            multiline='false'
+            multiline="false"
             allowedFormats={[]}
           />
         </header>
       }
       {!isExample &&
         <RichText
-          tagName='p'
-          className='page-section-description'
+          tagName="p"
+          className="page-section-description"
```

Comment:
I'm not sure which I understand the rule of when we need to use `"` and when `'`

Reply:
So we use double quotes("") for jsx attributes and single quotes('') for strings. So if you are declaring a variable that is a string you do something like `const myVar = 'This string'; `

For an attribute, it will be `<span className="some-class", data-attribute="some-value">My Span</span>;`